### PR TITLE
[stable/postgresql] Add external ip and port variables

### PR DESCRIPTION
**Pull request by tarasglek**
This lets one expose the service on external ip and a custom port number

### CODE REVIEW

File: stable/postgresql/templates/svc.yaml [hunk 1]
```diff
@@ -15,7 +15,15 @@ metadata:
...
+    {{ if .Values.port }}
+    port: {{ .Values.port }}
+    {{ else }}
     port: 5432
+    {{ end }}
     targetPort: postgresql
+  {{ if .Values.externalIP }}
+  externalIPs:
+    - {{ .Values.externalIP }}
+  {{- end }}
```
Comment (unguiculus) [inline stable/postgresql/templates/svc.yaml]: You should remove the if statements and add defaults to the values.yaml.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by linki** [comment]
@tarasglek do you want to sign the CLA so we can take in your changes?

**Comment by prydonius** [comment]
@tarasglek are you able to continue working on this?

**Comment by tarasglek** [comment]
@prydonius 
yes sorry, missed the earlier notifications that you guys were looking at this change. I'll submit the pullreq without conditionals as requested

**Comment by tarasglek** [comment]
@unguiculus I got rid of if statements.
`--set 'service.externalIPs=["192.168.50.4"],service.port=5433'`
works, but I'm not sure how one would pass multiple IPs since `,` is already used as a param separator

**Comment by unguiculus** [comment]
@tarasglek Here's an example for externalIPs: https://github.com/kubernetes/charts/blob/e60c6e9d6d0b3cf67291769a940a60f42521fdf5/stable/nginx-ingress/templates/controller-service.yaml#L18

**Comment by tarasglek** [comment]
@unguiculus thanks, that's similar to how I implemented it. Can I get this merged?

**Comment by unguiculus** [comment]
@tarasglek Well, the way you implemented it forces one to use the one-line format for a list, which is uncommon. I'd suggest you change it.

**Comment by tarasglek** [comment]
@unguiculus thanks for explaining. I'm new to yaml. This change is more inline with nginx example

**Comment by tarasglek** [comment]
Ping, can this be merged?

**Comment by prydonius** [comment]
@k8s-bot ok to test

**Comment by prydonius** [comment]
Thanks @tarasglek!!

**Pull request merged**